### PR TITLE
refactor: modify software category

### PR DESCRIPTION
**Pull request by zhhangBian**
# Contributing to LEMONADE reviews

Please review the project guidelines and check the boxes below.

**Subsections:**

- [x] Place papers only in **Level 3** or **Level 4** subsections.
- [x] Each subsection state the **common challenge**.
- [x] Each subsection has **at least 2 papers**.
- [x] Each subsection has **no more than 5 papers**.

**Tags:**

- [x] Tags are **specific techniques or contributions**
- [x] **No** broad area tags (like "performance") were used.
- [x] **No** vague feature tags (like "efficiency") were used *alone*.
- [x] Any potentially unclear acronyms in tags are **explained**.

### CODE REVIEW

**Review by youweizhuo — APPROVED** [review]
1. I just looked at the first few subsections. The group seems OK and we will fix them later every week.
2. The "common challenges" are not as good. The subsection titles are already self-explanatory so you don't have to put another sentence with almost the same words. For example: 




 # #### LLM Finetune
 # Finetune is a process of fine-tuning the LLM on a specific task.
 # This section includes the algorithms for LLM finetune.

These words are quite repetitive and there is no new information. What I want here is a summary of common challenges that these papers address. In the "LLM Quantization Methods" example, I think all of them are focusing on tradeoffs of accuracy and computation/memory. 

Anyways we will also fix them later when related sections are updated.
